### ISSUE TITLE
Difference between the downloaded model and trained model

### ISSUE THREAD

**Issue by HandsomeHehe** [guanyingc/PS-FCN#3]
Thank you for your code.
I train the model with the code and the test it in DiLiGent dataset according to the device . The mean error range is from 8.6° to 8.7°. 
Then I use the model you provided the result，and the result 8.39°。
It is same with the result in paper. I have no idea about this difference.
My GPU is 1080Ti.

**Comment by guanyingc** [comment]
Hi, thank you for your interests. There are many reasons (e.g., some random operations, different devices, library versions) that can affect the final results. I am not very sure about the exact reason.

The results in the paper is evaluated on the model trained before paper submission. The current code is released after some cleaning, but I have tried training the model again and the results are very close (around 8.40). However, your reproduced results are quite close although not perfect. 

Since we are training the model using synthetic dataset and test it on synthetic dataset, you may try use a smaller learning rate, or decay the learning rate a bit earlier, or train the model with fewer epochs (e.g., 25) to see if the results improve. The easiest way to verify the last strategy is to test the saved checkpoints from earlier epochs (e.g., 25). Hope this help.

**Comment by HandsomeHehe** [comment]
Thank you! I will try soon.

**Comment by guanyingc** [comment]
Okay. Please feel free to let me know if you have questions concerning the code.
